Components/Queues/QueueManager.php

```diff
@@ -133,7 +133,7 @@ public function getQueue(string $name, int $flag = AMQP_DURABLE, array $argument
 
     public function send(string $queueName, string $message, array $headers = [])
     {
-        $this->connect();
+//        $this->connect();
```

Comment:
Some particular reason of having this commented?

Reply:
The function connect is called many times among the publish request. I was doing a test to see if I remove the call of this function here might cause some impact and there was none.